test/tests/ver-1e/tests

```diff
@@ -1,12 +1,48 @@
 [Tests]
   design = 'Diffusion.md TimeDerivative.md DirichletBC.md'
   issues = '#12'
-  [csv]
+  [ver-1e]
     type = Exodiff
     input = ver-1e.i
-    exodiff = ver-1e_out.e
-    requirement = 'The system shall be able to model transient diffusion through a composite slab with a constant concentration boundary condition.'
+    exodiff = ver-1e_test_exodus.e
+    cli_args = 'Executioner/dt=0.2 Executioner/end_time=50 Outputs/exodus/file_base=ver-1e_test_exodus'
+    requirement = 'The system shall be able to model transient diffusion through a composite slab with a constant concentration boundary condition as the species source.'
     exodiff_opts = '-match_ids'
     map = False
```

Comment:
I honestly don't know what these do. Could you educate me?

Reply:
map = False makes results not match between parallel and serial runs. I'm not sure sure what match_ids does, but I think it's something similar. I don't know why this is needed, but didn't want to touch what I don't understand.